include/pybind11/attr.h

```diff
@@ -12,6 +12,9 @@
 
 #include "cast.h"
 
+// Include after pybind11 headers.
```

Comment:
This is always true, and we don't already have a note like this for each header. Let's see if we can enforce it with CI, and not write a comment in a single random header.

Reply:
Done.